lgr/char.py

```diff
@@ -234,6 +236,190 @@ def get_variants(self):
             for var in variants:
                 yield var
 
+    def get_index_cps(self, repertoire):
```

Comment:
Instead of providing the repertoire to a char I'll do conversely and call this from the repertoire, providing the char as an argument. This would keep the consistency with other methods.